frontend/src/components/WofArticleList.vue

```diff
@@ -0,0 +1,133 @@
+<template>
```

Comment:
This file looks completely unformatted. I have run vetur on this and got a bunch of changesi in indentation, and spaces between if/for and brackets. Can you make sure a formatter is run on this?